walkthroughs/week-4-tdd/intro/src/main/java/com/google/sps/Greeter.java

```diff
@@ -22,6 +22,15 @@
    * Returns a greeting for the given name.
    */
   public String greet(String name) {
-    return "Hello " + name;
+    String newName = name;
+    while(newName.contains(" ")){
```

Comment:
I think this method does exactly what you want already...

https://docs.oracle.com/javase/7/docs/api/java/lang/String.html#trim()